### DISCUSSION TITLE
Mac, ARM & x64 cant include library.

### DISCUSSION THREAD

**Discussion by Dariusz1989** [microsoft/vcpkg#25208] [Q&A]
Hey

I've recently learned that I need to have 2 separate binaries when making app on Mac M1-Intel. 
So In order to not lose my mind, I decided to have 2vcpkg's.
1. vcpkg
2. vcpkg_intel
![image](https://user-images.githubusercontent.com/13185728/173306490-12301ae7-e4e6-400b-86c7-863bdf716384.png)

Then I run a flag check in cmake to determine which vcpkg I want...

![image](https://user-images.githubusercontent.com/13185728/173306608-4814f75d-b9fb-4b3f-97d9-72c0839c562f.png)

And when I try to add library say
find_package(libzippp CONFIG REQUIRED)
I get he could not find a package configuration...
Now both of the vcpkgs have it and the only thing that I change (I think) is the vcpkg root... sooo how can I bite it? I tried passing explicit path to vcpkg/installed/x64-osx/share or lib or include/libzippp but even if he finds it he marks it as failed etc...

How can Include it?
TIA

ps1. I can find ARM package, but I cant find Intel... For some reason intel does not work :- (

**Comment by autoantwort** [comment]
Use the [manifest mode](https://github.com/microsoft/vcpkg/blob/master/docs/users/manifests.md) and the [cmake integration](https://github.com/microsoft/vcpkg/blob/master/docs/users/buildsystems/cmake-integration.md) and you don't have this problem.
